assignments/objects.js

```diff
@@ -19,27 +19,84 @@ const example = {
 
 // Write your intern objects here:
 
+const intern1 = {
+  "id": 1,
+  "name": 'Mitza',
+  "email": 'mmelloy0@psu.edu',
+  "gender": 'F'
+}
+
+const intern2 = {
+  "id": 2,
+  "name": 'Kennan',
+  "email": 'kdiben1@tinypic.com' ,
+  "gender": 'M',
+  "speak": function() {
+    console.log(`Hello, my name is ${this.name}`);
+   },
+  }
+
+
+const intern3 = {
+  "id": 3,
+  "name": 'Keven',
+  "email": 'kmummery2@wikimedia.org',
+  "gender": 'M'
+}
+
+const intern4 = {
+  "id": 4,
+  "name": 'Gannie',
+  "email": 'gmartinson3@illinois.edu',
+  "gender": 'M'
+}
+
+const intern5 = {
+  "id": 5,
+  "name": 'Antonietta',
+  "email": 'adaine5@samsung.com',
+  "gender": 'F',
+  "multiplyNums": function (a,b) {
+    return a * b;
```

Comment:
hmm, did it work for you?
Do you need space between `function (a,b)`?

Reply:
Yes it worked

Reply:
you are right! my extension removes this space automatically.

Reply:
http://crockford.com/javascript/code.html#function